templates/emails/email-order-details.php

```diff
@@ -33,7 +31,7 @@
 		$after  = '';
 	}
 	/* translators: %s: Order ID. */
-	echo wp_kses_post( $before . sprintf( __( '[Order #%s]', 'woocommerce' ) . $after . ' (<time datetime="%s">%s</time>)', $order->get_order_number(), $order->get_date_created()->format( 'c' ), wc_format_datetime( $order->get_date_created() ) ) );
+	echo wp_kses_post( $before . sprintf( esc_html__( '[Order #%s]', 'woocommerce' ) . $after . ' (<time datetime="%s">%s</time>)', $order->get_order_number(), $order->get_date_created()->format( 'c' ), wc_format_datetime( $order->get_date_created() ) ) );
```

Comment:
Maybe `esc_html__()` is not needed here since the whole string is being passed through `wp_kses_post()` before output? I'm not sure what is the best approach in this case and that is why I'm asking. I don't think double escaping can cause any issues but I'm not sure.

If we decide to change this line, we should change a few other lines where double escaping is also used.